lib/TraackrApi/Influencers.php

```diff
@@ -81,6 +116,40 @@ public static function lookupTwitter($username, $type = 'USERNAME')
         return $inf->get(TraackrApi::$apiBaseUrl . 'influencers/lookup/twitter/' . $username, $parameters);
     }
 
+    /**
+     * Add social account
+     *
+     * @param array $p
+     * @return bool|mixed
+     * @throws MissingParameterException
+     * @throws \UnexpectedValueException
+     */
+    public static function addSocial($p = array())
+    {
+        $inf = new Influencers();
+
+        $p = $inf->addCustomerKey($p);
+        $inf->checkRequiredParams($p, array('platform', 'customer_key'));
+
+        // Validate business requirements
+        $platform = $p['platform'];
+        if ('TWITTER' !== $platform && 'INSTAGRAM' !== $platform) {
+            throw new \UnexpectedValueException('Platform parameter must be "TWITTER" or "INSTAGRAM".');
+        }
+        if (empty($p['username']) && empty($p['user_id'])) {
+            throw new MissingParameterException("Either username or user_id must be present");
+        } else if (!empty($p['username']) && !empty($p['user_id'])) {
```

Comment:
Can just be `if` given the previous conditional throws an exception.

Reply:
that was a copy/paste but sure